src/test/kotlin/step3/fake/FakeNumberGenerator.kt

```diff
@@ -0,0 +1,15 @@
+package step3.fake
+
+import step3.model.NumberGenerator
+
+class NumberGeneratorGreaterThan4 : NumberGenerator {
+    override fun generate(): Int {
+        return 4
+    }
+}
```

Comment:
```suggestion
class FakeGenerator(private val number) : NumberGenerator {
    override fun generate(): Int {
        return number
    }
}
```

재활용 가능한 Fake 구조로 작성해봐도 좋을거같아요